src/includes/control.h

```diff
@@ -250,7 +252,7 @@ inline dasynq::rearm control_conn_cb(eventloop_t * loop, control_conn_watcher *
     }
     
     int watch_flags = 0;
-    if (!conn->bad_conn_close) {
+    if (!conn->bad_conn_close || conn->outbuf_size < 16384) {
```

Comment:
The condition is wrong, it should be `&&` not `||`.

Please use a named constant instead of a literal value.

Reply:
Done but I can't squash it now for some reasons, I will squash these 3 commits in merging process.

Reply:
How are you trying to squash? What is the problem?

Reply:
I used my phone to fix final things. I squashed them